### PR TITLE
Fix 72689

### PR DESCRIPTION
**Pull request by lola27**

### CODE REVIEW

File: src/app/helptext/jails/jails-add.ts [hunk 1]
```diff
@@ -531,6 +531,9 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
 individual devfs ruleset. Allow the creation of <i>tun</i> devices in \
 the jail.'),
 
+rtsold_placeholder: T('rtsold'),
```
Comment (Mrt134) [inline src/app/helptext/jails/jails-add.ts]:
```suggestion
rtsold_placeholder: T('rtsold With Autoconfigure IPv6'),
```

File: src/app/helptext/jails/jails-add.ts [hunk 2]
```diff
@@ -531,6 +531,9 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
...
+rtsold_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/jails/jails-add.ts]: Suggested help text:
```
Use
<a href="https://www.freebsd.org/cgi/man.cgi?query=rtsold" target="_blank">rtsold(8)</a>
as part of the IPv6 autoconfiguration process.
Sends ICMPv6 Router Solicitation messages to the interfaces to discover new routers.
```
Formatting might need adjustment.

File: src/app/helptext/jails/jails-edit.ts [hunk 3]
```diff
@@ -508,6 +508,9 @@ jail_zfs_mountpoint_tooltip: T('Enter the mountpoint for the \
 allow_tun_placeholder: T('allow_tun'),
 allow_tun_tooltip: T(''),
 
+rtsold_placeholder: T('rtsold'),
```
Comment (Mrt134) [inline src/app/helptext/jails/jails-edit.ts]:
```suggestion
rtsold_placeholder: T('rtsold With Autoconfigure IPv6'),
```

File: src/app/helptext/jails/jails-edit.ts [hunk 4]
```diff
@@ -508,6 +508,9 @@ jail_zfs_mountpoint_tooltip: T('Enter the mountpoint for the \
...
+rtsold_tooltip: T(''),
```
Comment (Mrt134) [inline src/app/helptext/jails/jails-edit.ts]: Same text as suggested above.

File: src/app/helptext/jails/jails-add.ts (hunk 1 above)
Comment (lola27) [reply src/app/helptext/jails/jails-add.ts]: Hi Tim, could you double check with Brandon about the placeholder changes? thanks

File: src/app/helptext/jails/jails-add.ts [hunk 5]
```diff
@@ -531,6 +531,11 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
 individual devfs ruleset. Allow the creation of <i>tun</i> devices in \
 the jail.'),
 
+rtsold_placeholder: T('Autoconfigure IPv6 With rtsold'),
```
Comment (wblock) [inline src/app/helptext/jails/jails-add.ts]: Title case:
```suggestion
rtsold_placeholder: T('Autoconfigure IPv6 with rtsold'),
```

File: src/app/helptext/jails/jails-add.ts [hunk 6]
```diff
@@ -531,6 +531,11 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
...
+rtsold_tooltip: T('Use <a href="https://www.freebsd.org/cgi/man.cgi?query=rtsold" \
+target="_blank">rtsold(8)</a> as part of the IPv6 autoconfiguration process. \
```
Comment (wblock) [inline src/app/helptext/jails/jails-add.ts]:
```suggestion
target="_blank">rtsold(8)</a> as part of IPv6 autoconfiguration. \
```

File: src/app/helptext/jails/jails-add.ts [hunk 7]
```diff
@@ -531,6 +531,11 @@ allow_tun_tooltip: T('Reveal <i>tun</i> devices for the jail with an \
...
+Sends ICMPv6 Router Solicitation messages to the interfaces to discover new routers.'),
```
Comment (wblock) [inline src/app/helptext/jails/jails-add.ts]:
```suggestion
Send ICMPv6 Router Solicitation messages to interfaces to discover new routers.'),
```

File: src/app/helptext/jails/jails-edit.ts [hunk 8]
```diff
@@ -508,6 +508,11 @@ jail_zfs_mountpoint_tooltip: T('Enter the mountpoint for the \
 allow_tun_placeholder: T('allow_tun'),
 allow_tun_tooltip: T(''),
 
+rtsold_placeholder: T('Autoconfigure IPv6 With rtsold'),
```
Comment (wblock) [inline src/app/helptext/jails/jails-edit.ts]: As above.

File: src/app/helptext/jails/jails-edit.ts [hunk 9]
```diff
@@ -508,6 +508,11 @@ jail_zfs_mountpoint_tooltip: T('Enter the mountpoint for the \
...
+rtsold_tooltip: T('Use <a href="https://www.freebsd.org/cgi/man.cgi?query=rtsold" \
+target="_blank">rtsold(8)</a> as part of the IPv6 autoconfiguration process. \
```
Comment (wblock) [inline src/app/helptext/jails/jails-edit.ts]: As above.

File: src/app/helptext/jails/jails-edit.ts [hunk 10]
```diff
@@ -508,6 +508,11 @@ jail_zfs_mountpoint_tooltip: T('Enter the mountpoint for the \
...
+Sends ICMPv6 Router Solicitation messages to the interfaces to discover new routers.'),
```
Comment (wblock) [inline src/app/helptext/jails/jails-edit.ts]: As above.

**Pull request merged**